### PR TITLE
Revise build.sh, add initial README.md

### PR DESCRIPTION
**Pull request by yf13**
This PR revises help message of build.sh so that to sync with the code.
It also adds an initial version of README file to help people get started quickly.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -145,30 +146,26 @@ function main() {
...
-    elif [ "$2" = "all" ]; then
-      build_linux
-      build_opensbi
-      build_uboot
-      build_rootfs
-      build_img
-    else
-      fault
-    fi
+  if [ "$1" = "linux" ]; then
```
Comment (RevySR) [inline build.sh]: Please do not modify the main function, this will destroy github action

Comment (yf13) [reply build.sh]: Very sorry about that.  

How do you think about also updating the `.github/workflow/build.yml` like below?
```
-             bash ./build.sh build all
+             bash ./build.sh all
```

Comment (RevySR) [reply build.sh]: Would like to keep the command categorized

Comment (yf13) [reply build.sh]: I see, will update the patch to stick to original usage.

**Review by RevySR — APPROVED** [review]

**Comment by RevySR** [comment]
https://github.com/ruyisdk/mkimg-k230-rv64ilp32/commit/9fafe78e4a076f2264a0e6d305180e5e392468dd
https://github.com/ruyisdk/mkimg-k230-rv64ilp32/commit/abed75c3224d7a173b49aeb38fa939b9eac87aeb

Merged.

**Pull request closed without merging**